packages/cosmic-swingset/src/launch-chain.js

```diff
@@ -89,27 +87,36 @@ async function buildSwingset(
   return { controller, mb, bridgeInbound, timer };
 }
 
-function computronCounter(limit, vatCost) {
-  assert.typeof(limit, 'bigint');
-  assert.typeof(vatCost, 'bigint');
-  let total = 0n;
+function computronCounter({
+  [BeansPerMaxBlock]: maxBlock,
```

Comment:
I'm probably missing something about destructuring assignment, but why does `BeansPerMaxBlock` need to be in square brackets?